keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -319,321 +295,20 @@ kind: BucketClass
 metadata:
   name: 
 provisioner: [1]
-supportedProtocols: {"azureblob", "gcs", "s3", ... } [2]
-accessMode: {"ro", "wo", "rw"} [3]
-releasePolicy: {"Delete", "Retain"} [4]
-bucketContentRef: [5]
-  name:
-  uuid:
-secretRef: [6]
-  name:
-  namespace:
+isDefaultBucketClass: [2]
```

Comment:
comment for `protocolAzure` above: it should be # 11 and swapped with `protocol`.